src/learn.jl

```diff
@@ -240,530 +240,6 @@ function evaluate!(predicted_hard_labels::AbstractVector,
-            plot_curve_data=(mean.(curve.bins), curve.fractions))
-end
-
-function _calculate_discrimination_calibration(predicted_hard_labels, votes;
-                                               class_of_interest_index)
-    elected_probabilities = _elected_probabilities(votes, class_of_interest_index)
-    bin_count = min(size(votes, 2) + 1, 10)
-    curve = calibration_curve(elected_probabilities,
-                              predicted_hard_labels .== class_of_interest_index; bin_count)
-    return (mse=curve.mean_squared_error,
-            plot_curve_data=(mean.(curve.bins), curve.fractions))
-end
-
-function _elected_probabilities(votes, class_of_interest_index)
-    elected_probabilities = Vector{Float64}()
-    for sample_votes in eachrow(votes)
-        actual_sample_votes = filter(v -> v in Set([1, 2]), sample_votes)
-        push!(elected_probabilities, mean(actual_sample_votes .== class_of_interest_index))
-    end
-    return elected_probabilities
-end
-
-function _calculate_voter_discrimination_calibration(votes; class_of_interest_index)
-    elected_probabilities = _elected_probabilities(votes, class_of_interest_index)
-
-    bin_count = min(size(votes, 2) + 1, 10)
-    per_voter_calibration_curves = map(1:size(votes, 2)) do i_voter
-        return calibration_curve(elected_probabilities,
-                                 votes[:, i_voter] .== class_of_interest_index;
-                                 bin_count=bin_count)
-    end
-
-    return (mse=map(curve -> curve.mean_squared_error, per_voter_calibration_curves),
-            plot_curve_data=map(curve -> (mean.(curve.bins), curve.fractions),
-                                per_voter_calibration_curves))
-end
-
-function _get_optimal_threshold_from_ROC(per_class_roc_curves; thresholds,
-                                         class_of_interest_index)
-    return _get_optimal_threshold_from_ROC(per_class_roc_curves[class_of_interest_index],
-                                           thresholds)
-end
-
-function _get_optimal_threshold_from_ROC(roc_curve, thresholds)
-    dist = (p1, p2) -> sqrt((p1[1] - p2[1])^2 + (p1[2] - p2[2])^2)
-    min = Inf
-    curr_counter = 1
-    opt_point = nothing
-    threshold_idx = 1
-    for point in zip(roc_curve[1], roc_curve[2])
-        d = dist((0, 1), point)
-        if d < min
-            min = d
-            threshold_idx = curr_counter
-            opt_point = point
-        end
-        curr_counter += 1
-    end
-    return collect(thresholds)[threshold_idx]
-end
-
-function _validate_threshold_class(optimal_threshold_class, classes)
-    has_value(optimal_threshold_class) || return nothing
-    length(classes) == 2 ||
-        throw(ArgumentError("Only valid for binary classification problems"))
-    optimal_threshold_class in Set([1, 2]) ||
-        throw(ArgumentError("Invalid threshold class"))
-    return nothing
-end
-
-"""
-    evaluation_metrics_row(observation_table, classes, thresholds=0.0:0.01:1.0;
```

Comment:
This is the only function that was NOT moved; it was deleted outright, and replaced by the (now-renamed) `refactored_evaluation_metrics_row` in src/metrics.jl.